packages/admin/src/Pages/Page.php

```diff
@@ -198,6 +200,11 @@ protected function getTitle(): string
             ->title();
     }
 
+    protected function getSubheading(): ?string
+    {
+        return static::$subtitle;
```

Comment:
I'm not sure about the naming of the property.

Reply:
Maybe rename it to "description" instead of subtitle/subheading

Reply:
Might be confusing with meta description. I think subheading could be suitable actually. Just not sure about the internal subtitle prop.

Reply:
Yeah this needs to be `$subheading`